### PR TITLE
[test-suite] rename a file

### PR DESCRIPTION
**Pull request by vbgl**
See https://github.com/coq/coq/pull/6218#issuecomment-428540277

### CODE REVIEW

**Comment by ppedrot** [comment]
I'll merge this when CI finishes.

**Review by ppedrot — APPROVED** [review]

**Comment by Zimmi48** [comment]
Don't wait for the full CI. The test-suite has passed already.

**Comment by ppedrot** [comment]
"And then a chain reaction of failure ensued." Not likely indeed, let's merge.

**Pull request merged**
